powershell-adapter/windowspowershell.resource.ps1

```diff
@@ -0,0 +1,165 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Do we need a separate script?  The previous version I believe had a parameter to know which PS to spawn and I think that was the only functional difference?

Reply:
not sure yet. even if they start the same, it will be easier to support long term if we seperate them. we might want to consider completely seperating them into two distinct modules/adapters. I can remove them if you prefer?

Reply:
It may make sense to separate once they start to diverge, but I think they are the same right now so it may be better to avoid confusion why there's two same scripts to maintain (like when we find bugs)